client/verta/tests/unit_tests/registry/test_model_dependencies.py

```diff
@@ -108,6 +119,7 @@ def test_class_module_names(dependency_testing_model) -> None:
     """ Verify that all expected module names are extracted as expected.
     """
     expected_modules = {
+        'builtins',
```

Comment:
Another std-lib package that gets caught by the new logic.  Ignored like the others.